### ISSUE TITLE
How does the log(Ko/w) influence the fraction unbound to Albumin ?

### ISSUE THREAD

**Issue by jonathanlsn** [Open-Systems-Pharmacology/Forum#706]
Hello,
I made two models for the same molecule on Mobi. The Only difference is the log(Ko/w) value I set to 4.7 for the first one and to 3.8 for the second one. I saw that the Fraction Unbound to Albumin was different either : 0.01 and 0.03 respectively.

I also tried to modify the parameter of the second model by setting the Fraction unbound to 0.01 (as the first one) and I obtain very different results. So I suppose this fraction unbound has a great influence on the simulation and I was wondering how does the log(Ko/w) influence in the calculation of the fraction unbound to Albumin ?

**Comment by StephanSchaller** [comment]
> I was wondering how does the log(Ko/w) influence in the calculation of the fraction unbound to Albumin ?

It doesn't.
Fraction unbound (fu) is a measured input parameter. But of course, log(Ko/w) and fu both influence the distribution of the molecule.

To learn more, I suggest to check out the literature on the basics of compartmental partitioning:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#adme-properties

Hope this helps.

Best,
Stephan

**Comment by jonathanlsn** [comment]
Indeed, I checked the equation for the calculation of fu and the Log(ko/w) doesn't appear but have an indirect influence on fu through the calculation of logD_pH_7p4.

Thank you for your help !

Best,
Jonathan
